### DISCUSSION TITLE
Is it possible to set different colors of resize handle?

### DISCUSSION THREAD

**Discussion by EtagiBI** [react-grid-layout/react-grid-layout#1702] [Q&A]
Hello, 
Widgets on my dashboard have different backgrounds (bright and dark). At the moment resize handle color is set to blackish, but it's almost impossible to see it on a dark background:
![изображение](https://user-images.githubusercontent.com/20855725/162439676-cba33690-878a-441d-9d56-c669484b9231.png)
![изображение](https://user-images.githubusercontent.com/20855725/162440168-a0d044b5-e795-4196-9e9e-2b59b5d840d2.png)

Is it possible to set different colors of resize handle for each type of widget? 

By the way, what's the purpose of using background image for resize handle?
![изображение](https://user-images.githubusercontent.com/20855725/162440495-2f7e6e04-e2c4-4f88-98eb-54862e3cdaa1.png)

**Comment by STRML** [comment]
Yeah, just key it on classes added to each widget or something else. 

Background image is just used so we don't have to actually embed a separate image. You could also do some trickery with pseudo elements and backgrounds or borders. Lots of ways to handle it.
